common/lib/xmodule/xmodule/tests/helpers.py

```diff
@@ -39,3 +39,17 @@ def get_current_user(self):
         user = XBlockUser()
         user.opt_attrs['edx-platform.username'] = 'bilbo'
         return user
+
+
+class StubAnonymousUserService(UserService):
```

Comment:
Instead of creating another class, you can add an `init()` to `StubUserService` which is passed the username and is_authenticated value and saves them as instance variables for use in `get_current_user(self)`.